src/ddscxx/include/org/eclipse/cyclonedds/domain/qos/DomainParticipantQosDelegate.hpp

```diff
@@ -36,11 +38,6 @@ namespace qos
 class OMG_DDS_API DomainParticipantQosDelegate
 {
 public:
-    DomainParticipantQosDelegate();
```

Comment:
I surprises me you can just throw these away, are you sure that can't cause any problems?

Reply:
Well, the copy constructor and destructor are in my eyes the definition of boilerplate, and can be completely handed off to the compiler.
By setting the default values of the members, which do not have default constructors (like the present_ flag on line 60), these values will then be passed on during construction when no other values are supplied.
This allows the user to skip the definition of the default constructor and just fall back on the default values supplied in the definition of the class.